client/less/classic-modal.less

```diff
@@ -0,0 +1,45 @@
+.success {
```

Comment:
My fear is that `.success` is too generic.

How about using something like `.challenges-success-modal`

Then just target the specific elements instead of using long chained selectors

`.challenges-success-modal .close`